File: 20250227/calculadora/Calculadora.java
public class Calculadora {
    public float valor=0;

    public Calculadora (float valor){
        this.valor = valor;
    }

    public void somar(float valor){
        this.valor += valor;
    }

    public void subtrair(float valor){
        this.valor -= valor;
    }

    public void multiplicar(float valor){
        this.valor *= valor;
    }    

    public void dividir(float valor){
        this.valor /= valor;
    }
}

File: 20250318/Calculadora.py
class Calculadora():

    def __init__(self, valorInicial ):
        self.valor=valorInicial

    def somar(self,operador):
        self.valor = self.valor+operador

    def subtrair(self, operador):
        self.valor = self.valor-operador
    
    def multiplicar(self, operador):
        self.valor = self.valor * operador

    def dividir(self, operador):
        try:
            self.valor = self.valor / operador
        except:
            print("Erro: não é possível realizar a operação indicada")

